service/src/test/java/uk/gov/hmcts/reform/fpl/model/DirectionResponseTest.java

```diff
@@ -0,0 +1,150 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.skyscreamer.jsonassert.JSONAssert;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
+
+import java.util.UUID;
+
+import static java.util.Collections.singletonList;
+import static java.util.UUID.randomUUID;
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.enums.DirectionAssignee.COURT;
+
+class DirectionResponseTest {
+    private static final ObjectMapper mapper = new ObjectMapper();
+
+    static {
+        mapper.findAndRegisterModules();
+    }
```

Comment:
Couldn't we use spring test and user the real jackson? That doesn't give us guarantee we are using the same config.